lib/active_admin/resource/routes.rb

```diff
@@ -42,8 +42,6 @@ def route_uncountable?
         config[:route_collection_name] == config[:route_instance_name]
       end
 
-      private
```

Comment:
In this case, it seems like the intention was to make the `RouteBuilder` class private, but it doesn't work like that. Should we add `private_constant :RouteBuilder` to make that actually happen?